src/arch/xtensa/xtos/CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ add_local_sources(xtos
 	user-vector.S
 	xea1/window-vectors.S
 	xea2/window-vectors.S
+	stub.c
 )
```

Comment:
@lkoenig is this taken from Xtensa initial sources or just made up? I try to understand if the naming and directory location most natural decision we can make.

Reply:
This is made up. I wasn't sure where to put the file. As this is OS specific I thought xtos would be suitable but I was not that convinced. I am very open to put it somwhere else.

Reply:
Ok, we could go with this for now. Just please fix the checkpatch.pl warnings.